<launch>
    <arg name="rviz_cloud" default="false" />
    <arg name="rviz_path" default="false" />

    <rosparam command="load" file="$(find publisher)/config/config.yaml" />

    <node pkg="publisher" type="publisher_node" name="publisher_node" output="screen" />

    <group if="$(arg rviz_cloud)">
        <node pkg="rviz" type="rviz" name="rviz" args="-d $(find publisher)/config/display_cloud.rviz" />
    </group>

    <group if="$(arg rviz_path)">
        <node pkg="rviz" type="rviz" name="rviz" args="-d $(find publisher)/config/display_odom.rviz"/>
    </group>

</launch>